### PR TITLE
Task page layout

### PR DESCRIPTION
**Pull request by aabdiaziz**
I have done built the page layout
1. navbar
2. footer 
3. social links
4. img background

I have also added my home page which inlcudes:
1. main img 
2.title & button

### CODE REVIEW

**Comment by charliecog** [comment]
General comments from looking at the front end:
1: hamburger menu expanded doesn't have the correct background colour and close box moves other content around when hovering over it
2: Facebook social icon should be png not jpeg as it currently has a visible white background
3: Same with hamburger menu icon
4: Social icons aren't clickable, link them to the relevant accounts
5: Github logo is the wrong dimensions and has stretched to make it look weird
6: The copyright symbol doesn't mean anything. 

Will look at the code now

**Comment by charliecog** [comment]
Extra general comments:
- delete unnecessary files or folders eg: full-stack.htm
- put images inside an image folder to keep your project files ordered and clean

File: style.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    }
+
+    #nav:target {
+        transform: translateX(340px);
+    }
+
+    .close {
+        text-align: right;
+        display: block;
+        text-decoration: none;
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply style.css]: so this cannot be seen. but it is located on the the hidden navbar which its function is to close the navbar after opened.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        font-size: 40px;
+        position: relative;
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply style.css]: taken out. thank you

File: style.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply style.css]: taken out

File: style.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        top: 0px;
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply style.css]: taken out

File: index.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply index.html]: done

File: style.css [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+        border: 3px bisque solid;
+    }
+
+    .close:hover {
+        margin-bottom: 0px;
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply style.css]: changed - removed

File: style.css [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+    }
+
+    .open {
+        text-align: left;
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply style.css]: changed - removed

File: index.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="icon" href="full-stack.png" type="image/x-icon">
+</head>
+<body>
+    <header>
+        <a class="toggle open" href="#nav"><img class="burger" src="burger.png" alt=""></a>
+        <navbar >
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply index.html]: done

File: index.html [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+            <ul id="nav">
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply index.html]: done

**Pull request merged**